ui/app/styles/components/namespace-picker.scss

```diff
@@ -5,65 +5,33 @@
 
 .namespace-picker {
   position: relative;
-  color: $white;
+  color: var(--token-color-palette-neutral-300);
   display: flex;
-  fill: $white;
   padding: $spacing-xxs $spacing-xs;
   width: 100%;
-
-  @include from($mobile) {
-    margin-left: -$spacing-xs;
-    padding: $spacing-xxs 0 $spacing-xxs $spacing-s;
-    width: auto;
-  }
 }
 
 .namespace-picker.no-namespaces {
   border: none;
   padding-right: 0;
 }
+
 .namespace-picker-trigger {
   align-items: center;
   display: flex;
   flex: 1 1 auto;
   height: 2rem;
   justify-content: space-between;
-  padding: 0;
-  text-align: left;
-
-  @include from($mobile) {
-    height: auto;
-    padding: $spacing-xs $spacing-m;
-  }
-
-  .is-status-chevron {
-    transform: rotate(-90deg);
-
-    @include from($mobile) {
-      transform: rotate(0deg);
-    }
-  }
-
-  &.ember-basic-dropdown-trigger--below .is-status-chevron {
-    transform: rotate(0deg);
-
-    @include from($mobile) {
-      transform: rotate(180deg);
-    }
-  }
+  margin-right: $spacing-xxs;
 }
+
 .namespace-name {
-  display: inline-block;
-  flex: 1 1 auto;
   font-size: 1rem;
-  margin: 0 $spacing-xs;
-
-  @include from($mobile) {
-    margin-left: $size-10;
-  }
+  margin-left: $spacing-xs;
```

Comment:
I'm being a little pedantic here, but can we use a helper if we have one for the margin situation? We also may have a font-size helper as well. If it's a pain, ignore me, just pushing helper usage instead of component specific styling when I can.

Reply:
I didn't want to get too in the weeds with removing classes in favor of helpers since there is already a namespace stylesheet. I'm not sure if we have fully fleshed out the plan for updating styling post bulma yet. FWIW it doesn't look like there is a 1rem font size helper class.